lib/cocoapods/command/update.rb

```diff
@@ -42,6 +44,14 @@ def initialize(argv)
           end
         end
 
+        unless skipped_pods.empty?
+          if @pods
+            @pods.delete_if { |pod| skipped_pods.include?(pod) }
+          else
+            @pods = config.lockfile.pod_names.reject { |pod| skipped_pods.include?(pod) }
```

Comment:
should probably raise if you're skipping a pod that isn't in the lockfile

Reply:
@segiddins added and informative error in case the user is trying to skip a pod that is not installed. Let me know what you think.